.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -142,7 +142,9 @@ jobs:
 
           CIBW_TEST_REQUIRES: pytest pytest-cov pytest-mock flaky
 
-          CIBW_BEFORE_TEST: if ${{ matrix.pl_backend == 'lightning_kokkos'}}; then SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" pip install -e . -vv; fi
+          CIBW_BEFORE_TEST: |
+            python -m pip install git+https://github.com/PennyLaneAI/pennylane.git@master
+            if ${{ matrix.pl_backend == 'lightning_kokkos'}}; then SKIP_COMPILATION=True PL_BACKEND="lightning_qubit" pip install -e . -vv; fi
```

Comment:
So if the backend installed in LK, then we must also install LQ but not build the module? 
This seems a little convoluted to me. I wonder if there is a clearer way.

Should we always ensure LK installs LQ as well, so that we never have a scenario where the package is installed without its required dependencies?

Reply:
Hi @mlxd, this was not a new addition. This is the patch we have in place to run the LKokkos tests and wheels.
A more definite solution will come when we have a proper installation for LKokkos.
The strategy I have in mind to install LKokkos will actually be moving only the LKokkos-specific files and having as a pre-requisite the LQubit installation (with core files) targeting the same version.